README.rst

```diff
@@ -144,6 +154,13 @@ to the IGNORE\_CEC list in PyChromecast like in the example below.
     pychromecast.IGNORE_CEC.append('*')  # Ignore CEC on all devices
     pychromecast.IGNORE_CEC.append('Living Room')  # Ignore CEC on Chromecasts named Living Room
 
+Firewall Rules
+-----------------
+MDNS discovery takes place on port UDP 5353.
+SSDP discovery takes place on port UDP 1900.
+
+You may want\need to create a firewall rule that allows both of these ports both inbound and outbound. Device discovery may not function properly until this rule is in place.
+
```

Comment:
```suggestion
Networking requirements
-----------------
Pychromecast relies on mDNS to discover cast devices. The mDNS protocol relies on multicast UDP on port 5353 which comes with several implications for discovery to work:
- Multicast UDP must be forwarded by WiFI routers; some WiFi routers are known to drop multicast UDP traffic.
- The device running pychromecast must allow both inbound and outbound traffic on port 5353.
- The device running pychromecast must be on the same subnet as the cast devices because mDNS packets are not routed across subnets.

If not all of these conditions are met, discovery will not work. In cases where these conditions are impossible to meet, it's possible to pass a list of known IP-addresses or host names to the discovery functions.

```